client/src/components/StreamField/blocks/FieldBlock.js

```diff
@@ -27,7 +27,7 @@ export class FieldBlock {
       // eslint-disable-next-line no-console
       console.error(e);
       this.setError([
-        ['This widget failed to render, please check the console for details']
+        { messages: ['This widget failed to render, please check the console for details'] }
```

Comment:
Sorry, I know this faking of a ValidationError is a bit hacky (and likewise the duplication of the ValidationError definition in the tests), but making a single ValidationError definition visible to both the StreamField components and the wagtailadmin/js/telepath/widgets.js endpoint would have involved more code reshuffling than I'm happy to do post-RC...